### ISSUE TITLE
Add "restart R?" yesno option when running `install_greta_deps()`

### ISSUE THREAD

**Issue by njtierney** [greta-dev/greta#523]
* only shows up when `interactive()` returns TRUE (to avoid issues with scripting and restarting R without asking)
* or potentially add a `restart = FALSE` option in `install_greta_deps()` instead/in addition?

**Comment by njtierney** [comment]
Something like:

```
Would you like to restart R now: (yesno options)
```

with the restart R option, it could even have something that restarts R and runs `library(greta)`

```r
.rs.restartR(afterRestartCommand = "library(greta)")
```

**Comment by njtierney** [comment]
Closed in #672
